scripts/generate-google3-docs/resource-reference/generated/resource-docs/iam/iampolicymember.md

```diff
@@ -1091,7 +1091,7 @@ metadata:
   name: iampolicymember-dep-orgrole
 ```
 
-### Policy Member With BigQueryConnection Reference
+### Policy Member With Bigqueryconnection Reference
```

Comment:
qq: Why do we change this? It should be BigQueryConnection or BigQuery Connection

Reply:
cc @jingyih who is our expert for BigQuery resource groups.